orc8r/cloud/go/services/orchestrator/obsidian/models/conversion.go

```diff
@@ -626,6 +627,21 @@ func (m *GatewayVpnConfigs) FromBackendModels(ctx context.Context, networkID str
 	return nil
 }
 
+func LastGatewayCheckInWasRecent(gatewayStatus *GatewayStatus, magmadConfig *MagmadGatewayConfigs) bool {
+	if gatewayStatus == nil {
+		return false
+	}
+	checkinTime := time.UnixMilli(int64(gatewayStatus.CheckinTime))
+
+	var checkinInterval time.Duration
+	if magmadConfig != nil {
+		checkinInterval = time.Duration(int64(magmadConfig.CheckinInterval)) * time.Second
+	} else {
+		checkinInterval = time.Minute
```

Comment:
Using the default interval if no config can be found.

Reply:
Can you create a constant, like `DEFAULTCHECKININTERVAL` and set it on top of this file?

Reply:
Done.